python/graphstorm/config/argument.py

```diff
@@ -541,34 +541,34 @@ def save_model_path(self):
         return None
 
     @property
-    def save_model_per_iters(self):
```

Comment:
You also need to update --save-model-per-iters

Reply:
Also also check if any test scripts use --save-model-per-iters

Reply:
Not following. I did replace `save_model_per_iters` with `save_model_frequency`.